specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2022-09-01/containerInstance.json

```diff
@@ -0,0 +1,2402 @@
+          "format": "int32"
+        },
+        "periodSeconds": {
+          "description": "The period seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "failureThreshold": {
+          "description": "The failure threshold.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "successThreshold": {
+          "description": "The success threshold.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "timeoutSeconds": {
+          "description": "The timeout seconds.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "ContainerGroup": {
+      "description": "A container group.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "$ref": "#/definitions/ContainerGroupProperties"
+        }
+      ]
+    },
+    "ContainerGroupProperties": {
+      "description": "The container group properties",
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/ContainerGroupIdentity",
+          "description": "The identity of the container group, if configured."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "description": "The container group properties",
+          "properties": {
+            "provisioningState": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The provisioning state of the container group. This only appears in the response."
+            },
+            "containers": {
+              "type": "array",
+              "description": "The containers within the container group.",
+              "items": {
+                "$ref": "#/definitions/Container"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            },
+            "imageRegistryCredentials": {
+              "type": "array",
+              "description": "The image registry credentials by which the container group is created from.",
+              "items": {
+                "$ref": "#/definitions/ImageRegistryCredential"
+              },
+              "x-ms-identifiers": [
+                "username"
+              ]
+            },
+            "restartPolicy": {
+              "type": "string",
+              "description": "Restart policy for all containers within the container group. \n- `Always` Always restart\n- `OnFailure` Restart on failure\n- `Never` Never restart\n",
+              "enum": [
+                "Always",
+                "OnFailure",
+                "Never"
+              ],
+              "x-ms-enum": {
+                "name": "ContainerGroupRestartPolicy",
+                "modelAsString": true
+              }
+            },
+            "retryTimeoutMinutes": {
```

Comment:
Does this have a default value? Will the RP  add a default value even if not in the request? I know our current is 30 minutes, but not sure if that is going to be set in this property automatically.